### ISSUE TITLE
examples/coil_sensitivity_maps improvements

### ISSUE THREAD

**Issue by KrisThielemans** [CCPPETMR/xGadgetron#12]
do we need to use the RemoveROOversamplingGadget gadget here? I'd prefer to use a higher level method.

@DANAJK or @ckolbPTB would you be able to write some documentation in here regarding the difference between CoilImages and CoilSensitivityMaps? Ideally also something about the SSRS setting (do we need it here? If not, I'd mention in the doc that it is optional).

There is no MATLAB version of this demo yet.

**Comment by ckolbPTB** [comment]
I have started to add comments to the file but I don't fully understand it. 

The line "from ismrmrdtools import coils, show" suggests to me that ismrmrd->ismrmrd-python-tools is required. On the github site I only see that ismrmrd->ismrmrd is required. Am I missing something?

**Comment by KrisThielemans** [comment]
correct. It seems that this demo is using
`ismrmrd-python-tools.coils.calculate_csm_inati_iter` and `ismrmrd-python-tools.coils.calculate_csm_walsh`. 
These are independent of Gadgetron coil stuff. It's also unclear to me how this differs from our `CoilSensitivityMaps`. My guess is that the demo compares that with the alternative implementations, which makes sense. I guess would then make this an optional dependency on ismrmrd-python-tools. @DANAJK, can you help?

Note that we won't have a MATLAB equivalent of this part of the demo.

The demo also uses `ismrmrd-python-tools.show.imshow`. @evgueni-ovtchinnikov, is this any different from `matplotlib.imshow`?
 
finally, I also so `CIs.as_ndarray`. @evgueni-ovtchinnikov, should/can this be `CIs.as_array`?

**Comment by evgueni-ovtchinnikov** [comment]
Yes ismrmrd-python-tools.show.imshow is different from matplotlib.imshow.

CIs.as_ndarray: now renamed, please pull update.

**Comment by KrisThielemans** [comment]
any reason to choose ismrmrd-python-tools.show.imshow over matplotlib.imshow? Would reduce dependencies...

**Comment by evgueni-ovtchinnikov** [comment]
Can matplotlib.show do what ismrmrd-python-tools.show.imshow does? It does not look so to me (see show.py attached.)
